numpy/add_newdocs.py

```diff
@@ -5884,7 +5891,24 @@ def luf(lamdaexpr, *args, **kwargs):
     >>> np.add.reduce(X, 2)
     array([[ 1,  5],
            [ 9, 13]])
-
+           
+    You can use the initial argument to initialize the reduction with a
```

Comment:
Maybe put double-backquote around `initial` - otherwise "initial argument" reads as if the first argument is meant.

Or avoid it altogether with:
```
You can also start the reduction with a different initial value.
```